### DISCUSSION TITLE
Reorder calendars on the settings page

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [davish/obsidian-full-calendar#433] [Feature Requests]
When in the settings have the option to reorder them so they can be customized when adding new events.

**Comment by davish** [comment]
Thanks for the feature request! Right now, the workaround is to delete calendars and re-add them in the right order, but that is certainly cumbersome.

I'm planning on a frontend/settings overhaul at some point as I go through and improve the code organization for a 1.0 release, so I'll try and tackle this as part of that.
